### ISSUE TITLE
Doesn't seem to be listening to private conversation

### ISSUE THREAD

**Issue by raine** [huafu/hubot-gitter2#14]
The bot works in a room but doesn't respond when using commands privately.

Nothing seems to appear in the log either.

**Comment by purplefish32** [comment]
Usually you have to restart the bot when you subscribe to a new channel, otherwise it wont answer

**Comment by huafu** [comment]
Yup AFAIR @purplefish32 has the right answer, will see if I can get some time in the near future to work on this, tho I am not using Gitter anymore, switched to Slack...
